### PR TITLE
Protocol README and size recommendation

### PR DESCRIPTION
**Pull request by afrittoli**
Add a README to the vocabulary-draft folder.
Add links from the main README to the vacabulary, sdk and poc folders.
Add notes about lightweight events, as agreed in https://github.com/cdfoundation/sig-events/discussions/47

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

### CODE REVIEW

**Review by salaboy — APPROVED** [review]

File: vocabulary-draft/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Continuous Delivery Events
+
+Continuous Delivery Events (CD Events or CDE) is a common event format for the Continuous Delivery (CD) ecosystem.
```
Comment (m-linner-ericsson) [inline vocabulary-draft/README.md:3]: Do we want to have something else here like.
> We see these events as an enabler for interoperability between different CI/CD tools as the communication is tool agnostic

**Review by m-linner-ericsson — COMMENTED** [review]
Looks good, just felt a bit thin in the introduction but maybe we should have such things here.

File: vocabulary-draft/README.md (hunk 1 above)
Comment (afrittoli) [reply vocabulary-draft/README.md:3]: Thanks @m-linner-ericsson - I will add it.

**Review by m-linner-ericsson — APPROVED** [review]

**Pull request merged**